### PR TITLE
Allow asserting user existence by username

### PR DESCRIPTION
**Pull request by kane-c**

### CODE REVIEW

File: ixprofile_client/steps.py [hunk 1]
```diff
@@ -175,8 +175,10 @@ def no_user_on_profile_server(self):
     """
 
     for row in self.hashes:
-        details = webservice.profile_server.find_by_email(row['email'])
-        assert not details, "User present: %s" % row['email']
+        attr = 'email' if 'email' in row else 'username'
+        details = getattr(webservice.profile_server,
+                          'find_by_%s' % attr)(row[attr])
```
Comment (koterpillar) [inline ixprofile_client/steps.py]: Do we want to break greppability here?

File: ixprofile_client/features/functional_example.feature [hunk 2]
```diff
@@ -15,6 +15,9 @@ Feature: Integrate with Profile Server
 	    And I do not have users in the fake profile server:
 	      | email                   |
 	      | new_user@profile.server |
+        And I do not have users in the fake profile server:
```
Comment (koterpillar) [inline ixprofile_client/features/functional_example.feature]: Indentation

Comment (kane-c) [inline ixprofile_client/features/functional_example.feature]: Argh tabs strike again.

**Comment by koterpillar** [comment]
++ when passes

**Pull request merged**